tests/MyCSharp.HttpUserAgentParser.AspNetCore.UnitTests/MyCSharp.HttpUserAgentParser.AspNetCore.UnitTests.csproj

```diff
@@ -4,6 +4,10 @@
         <OutputType>Exe</OutputType>
     </PropertyGroup>
 
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
```

Comment:
Why this change?

Reply:
I will transfer the Common Definitions we Roslyn etc into the Build Props... then I have to separate them here anyway.

Reply:
like this 
https://github.com/mycsharp/portal/blob/8e0ce63df561cda5d0f30fe0ab1346e606879f88/Directory.Build.props#L61

Reply:
Ah, OK. But a separate PR for this would be welcomed 😉 That change is unrelated to "remove fluent assertions".

Reply:
Yes, thats the plan